server/schema/customer-order/res.js

```diff
@@ -1,12 +1,21 @@
 import {_auth} from '../../util'
-import {Customer} from '../../models'
+import {Customer, Order} from '../../models'
 const resolvers = {
   RootQuery: {
     async fetchCustomerOrders(_, {input}, {loggedInUser}) {
       _auth(loggedInUser)
       let customer = new Customer({id: input})
       return await customer.getOrders({order: [['createdAt', 'DESC']]})
     },
+    async fetchCustomerOrderDetail(_, {input}) {
+      let order = new Order({id: input})
+      let placeOrderMethod = await Order.findOne({where: {id: input}})
```

Comment:
placeOrderMethod and order  are actually the same ?
-> why need to create 2 separated variables ?

Reply:
placeOrderMethod.getOrderDetails()  -> because placeOrderMethod is actually an Order object
so why need to call getOrderDetails() from order, while we already have placeOrderMethod ??